File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/creating-an-association-between-entities_3.vb
    Public Shared Function ContactToSalesOrderDetail(ByVal contactID As Integer) As IEnumerable(Of SalesOrderDetail)
        Const ServerName As String = "MySQLServerName"
        Dim dataContext As AdventureWorksDataContext = _
            New AdventureWorksDataContext("Data Source=" & ServerName & _
                ";Initial Catalog=AdventureWorks;Integrated Security=True")

        Dim orderList As IEnumerable(Of SalesOrderDetail) = _
            From orders In dataContext.SalesOrderHeaders _
            Join orderDetails In dataContext.SalesOrderDetails On _
                orders.SalesOrderID Equals orderDetails.SalesOrderID _
            Where orders.ContactID = contactID _
            Select orderDetails
        Return orderList

    End Function

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/creating-an-association-between-entities_3.cs
        public static IEnumerable<SalesOrderDetail> ContactToSalesOrderDetail(int contactID)
        {
            const string ServerName = "MySQLServerName";
            AdventureWorksDataContext dataContext = new AdventureWorksDataContext
                  ("Data Source=" + ServerName + ";" +
                   "Initial Catalog=AdventureWorks;Integrated Security=True");

            IEnumerable<SalesOrderDetail> orderList =
                from orders in dataContext.SalesOrderHeaders
                join orderDetails in dataContext.SalesOrderDetails on
                    orders.SalesOrderID equals orderDetails.SalesOrderID
                where orders.ContactID == contactID
                select orderDetails;
            return orderList;
        }